modules/trusted-firmware-m/tfm_boards/common/assert.c

```diff
@@ -24,7 +24,7 @@
 	"\nRefer to the documentation section 'TF-M partition alignment requirements'" \
```

Comment:
The MPC handles this on nrf54l, so it may be a bit misleading to refer to spu requirements.

Reply:
If you have any feed back on this assert please use the following PR to provide it: 

> PR https://github.com/nrfconnect/sdk-nrf/pull/15210 needs to be merged before this and I'll rebase it on-top when that's done. 

This commit is not intended to be included in this PRs final form. It's to be rebased on-top of the changes to this but that PR was stale which is why I posted with a workaround.

Reply:
> Currently waiting for the fix found in PR https://github.com/nrfconnect/sdk-nrf/pull/15210 so this is a work
around since the alignment requirements is wrong for NRF54LX.